src/YoTennis/Controllers/MatchController.cs

```diff
@@ -28,6 +28,47 @@ public MatchController(UserManager<ApplicationUser> userManager, IMatchListServi
             _matchListService = matchListService;
         }
 
+        public async Task<IActionResult> Add()
```

Comment:
I'd suggest to rename to something like `AddPastMatch`

Reply:
Done